Jenkinsfile

```
@@ -134,16 +134,23 @@ pipeline {
     post {
         success {
             script {
-                if (env.GIT_BRANCH != 'master') {
-                    notify("${NOTIFICATION_CHANNEL}", ":white_check_mark: Branch $env.GIT_BRANCH PR $env.CHANGE_URL deployed by $env.CHANGE_AUTHOR_EMAIL via $env.BUILD_URL and available at https://$env.CFGOV_HOSTNAME.")
+                if (env.GIT_BRANCH != 'master' or env.GIT_BRANCH != 'main') {
+                    notify("${NOTIFICATION_CHANNEL}", ":white_check_mark: PR $env.CHANGE_URL deployed by $env.CHANGE_AUTHOR_EMAIL via $env.BUILD_URL and available at https://$env.CFGOV_HOSTNAME.")
```

I think we want this to be just `CHANGE_AUTHOR`. It's currently just spitting out `null`s.

```suggestion
                    notify("${NOTIFICATION_CHANNEL}", ":white_check_mark: PR $env.CHANGE_URL deployed by $env.CHANGE_AUTHOR via $env.BUILD_URL and available at https://$env.CFGOV_HOSTNAME.")
```

---

I've made this fix but I think we may wind upw ith this spitting out null as well, if so I'll just simplify the notification